### PR TITLE
Budget app

### PR DESCRIPTION
**Pull request by binyamolango**
# Hi Reviewer 👋
Below is a list of the accomplished tasks while working on this milestone:

- Set up the Git repository for version control.
- Created the Rails application with the necessary tools and dependencies.
- Utilized Postgres as the database, as specified in the requirements.
- Created User, Category, and Record models based on the provided ERD.
- Added associations between User, Category, and Record models.
- Implemented validations for mandatory fields in the models.
- Integrated Devise for user authentication.
- Configured Devise and generated the User model.
- Implemented authentication filters to control access to pages.
- Ran migrations to create tables for User, Category, Record, and CategoryRecord.
- Implemented a simple splash screen with the app name and links to sign-up and sign-in pages.
- Generated controllers and define their actions
- Implemented the required views.
- Styled the application according to the required design.
- Wrote unit tests for model validations, requests, and integration tests.
- Fixed linters' errors.
- Deployed the application on the Render platform.
- Recorded a [Video](https://www.loom.com/share/2e3733a09bd74f9095e58694303d1287?sid=8d2b1164-2c34-45ca-bb5e-8b1495426c13).

### CODE REVIEW

File: app/views/categories/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    	</div>
+    	<div>
+      	<%= form.label :icon %><br>
+      	<%= form.text_field :icon, placeholder: "Icon link", class: "field", required: true %>
+    	</div>
+			<div class="add-item">
+    		<%= form.submit "ADD CATEGORY", class: "add" %>
+			</div>
+  	<% end %>
+	</div>
```
Comment (thecodechaser) [inline app/views/categories/new.html.erb:22]: - [ ] When users create a category successfully the users should redirected to the homepage(categories page) as requested in the project requirements. Currently, users are redirected to the transaction page of the created category. Please make sure to fix this issue.

image

File: README.md [hunk 2]
```diff
@@ -118,11 +128,6 @@ To run tests, run the following command:
 - GitHub: [@binyamolango ](https://github.com/binyamolango)
 - LinkedIn: [Binyam Yohannes](https://www.linkedin.com/in/binyamyohannes/)
 
-👤 **Muhammad Bilal Liaquat**
-
-- GitHub: [@muhammadbilal ](https://github.com/BilalLiaquat7)
-- LinkedIn: [Muhammad Bilal](https://www.linkedin.com/in/muhammad-bilal-liaquat-87863390/)
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (thecodechaser) [inline README.md:133]: - [ ] In the Acknowledgments section of the README file, Please make sure that you give appropriate credit to the author [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for the original design idea as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @binyamolango👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## To Highlight 🎉
- All tests are passed. ✔️ 
- Good job with model validations. ✔️
- Nice UI design. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @binyamolango 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Acknowledged the author of the design used ✔️ 
Implemented back button that redirects to the homepage ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**